Incompatible old client dependency

I have been using 1.0.2 of this package along with `feathers-authentication-client` 0.3.1, and recently updated to the latest 1.2.1.

My Google authentication strategy broke with the upgrade, and I think I've managed to narrow it down to #433 that added an older (perhaps incompatible?) version of `feathers-authentication-client` 0.1.10.

Do you know why an older version is referenced?

@marshallswain Are you implying that 0.1.10 is a more recent version than 0.3.1?

@daffl That seems to be the fixed required, but somehow I don't seem to be able to test it.  I updated `package.json` with:

```
-    "feathers-authentication": "^1.0.2",
-    "feathers-authentication-client": "^0.3.1",
+    "feathers-authentication": "git://github.com/feathersjs/feathers-authentication.git#a77b557942a4ef1f1647ed85fd4ab51f88902fe5",
````

and despite having the correct package in `node_modules`, I end up with the following when trying to execute `node /webpack/output/file.js`

```
module.js:472
    throw err;
    ^

Error: Cannot find module 'feathers-authentication'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/webpack/output/file.js:148:18)
    at __webpack_require__ (/webpack/output/file.js:20:30)
```

Can you provide pointers on how I might be able to test #480?

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
   },
   "dependencies": {
     "debug": "^2.2.0",
-    "feathers-authentication-client": "^0.1.10",
+    "feathers-authentication-client": "^0.3.1",
     "feathers-commons": "^0.8.4",
     "feathers-errors": "^2.4.0",
     "feathers-socket-commons": "^2.3.1",

